fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraBackupIT.java

```diff
@@ -85,4 +86,36 @@ public void shouldRoundTripBackups() throws Exception {
         // check that we made it
         assertEquals(OK.getStatusCode(), getStatus(new HttpGet(serverAddress + objName)));
     }
+
+    @Test
+    public void backupNotFromRoot() throws Exception {
+        final String objName = randomUUID().toString();
+
+        // Create object
+        createObjectAndClose(objName);
+        // Create datastream
+        createDatastream(objName, "testDS", text);
+        // Verify object exists
+        assertEquals(OK.getStatusCode(), getStatus(new HttpGet(serverAddress + objName)));
+
+        // back it up from the object
+        final File requestedDir = createTempDir();
+        logger.debug("Backing up repository to {}", requestedDir.getCanonicalPath());
+        final HttpPost backupObjectRequest = new HttpPost(serverAddress + objName + "/fcr:backup");
+        backupObjectRequest.setEntity(new StringEntity(requestedDir.getCanonicalPath()));
+
+        try (CloseableHttpResponse backupResponse = execute(backupObjectRequest)) {
+            assertEquals(BAD_REQUEST.getStatusCode(), getStatus(backupResponse));
```

Comment:
Where? How about other checking the status code in this file, are they all need to be fixed?